packages/lambda-functions/functions/hello-world/app.ts

```diff
@@ -1,22 +1,117 @@
+import { PrismaClient } from '../../../../packages/db/src/index';
```

Comment:
Lambda wasn't recognizing the `@translation/db` alias from `tsconfig.base.json`, so I used this instead.

But now I get this error when I try to run the lambda function: `ENOENT: no such file or directory, open '/var/task/schema.prisma'`

I think this is what happens when it tries to access the prisma schema from inside the docker container.